webapp/packages/plugin-administration/src/ConfigurationWizard/ServerConfiguration/Form/ServerConfigurationInfoForm.tsx

```diff
@@ -1,43 +1,67 @@
 
-import type { IServerConfigurationPageState } from '../IServerConfigurationPageState.js';
 import { MIN_SESSION_EXPIRE_TIME } from './MIN_SESSION_EXPIRE_TIME.js';
+import type { ServerConfigurationFormState } from '../ServerConfigurationFormState.js';
+import { getServerConfigurationFormPart } from '../getServerConfigurationFormPart.js';
+import { useAvailableHosts } from './useAvailableHosts.js';
 
 interface Props {
-  state: IServerConfigurationPageState;
+  formState: ServerConfigurationFormState;
 }
 
-export const ServerConfigurationInfoForm = observer<Props>(function ServerConfigurationInfoForm({ state }) {
+export const ServerConfigurationInfoForm = observer<Props>(function ServerConfigurationInfoForm({ formState }) {
   const serverConfigLoader = useResource(ServerConfigurationInfoForm, ServerConfigResource, undefined);
+  const part = getServerConfigurationFormPart(formState);
+  const state = part.state;
   const translate = useTranslate();
+  const availableHostsState = useAvailableHosts(formState);
 
   return (
     <Group form gap>
       <GroupTitle>{translate('administration_configuration_wizard_configuration_server_info')}</GroupTitle>
+      <Switch
+        name="secureCookies"
+        state={state.serverConfig}
+        description={translate('administration_configuration_wizard_configuration_secure_cookies_description')}
+        mod={['primary']}
+        small
+      >
+        <div className="tw:flex tw:items-center tw:gap-1.5">
+          {translate('administration_configuration_wizard_configuration_secure_cookies')}
+          {!state.serverConfig.secureCookies && (
+            <IconOrImage
+              title={translate('administration_configuration_wizard_configuration_secure_cookies_warning')}
+              icon="/icons/warning_icon.svg"
+              width={24}
+            />
+          )}
+        </div>
+      </Switch>
       <InputField type="text" name="serverName" state={state.serverConfig} required medium>
         {translate('administration_configuration_wizard_configuration_server_name')}
       </InputField>
-      <InputField
+      <Textarea
         title={translate('administration_configuration_wizard_configuration_server_url_description')}
-        type="url"
-        name="serverURL"
-        state={state.serverConfig}
+        name="availableHosts"
+        rows={3}
+        state={availableHostsState}
         readOnly={serverConfigLoader.resource.distributed}
-        required
-        medium
+        description={translate('administration_configuration_wizard_configuration_available_hosts_description')}
+        placeholder={`example.com\nlocalhost:5000\n127.0.0.1`}
+        onChange={availableHostsState.onAvailableHostsChange}
       >
-        {translate('administration_configuration_wizard_configuration_server_url')}
-      </InputField>
+        {translate('administration_configuration_wizard_configuration_available_hosts')}
```

Comment:
if https://github.com/dbeaver/pro/issues/5241 merges first, we need to move this child to label prop